core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -11944,7 +11944,16 @@ private void queryTimestampsInRange(Connection connection) throws SQLException,
                 ts.setNanos((int) ((micros % 1_000_000) * 1000));
                 statement.setTimestamp(1, ts);
                 statement.setTimestamp(2, ts);
-                statement.executeQuery();
+
+                if (!legacyMode) {
```

Comment:
Wouldn't it be simpler to close the result set properly on each iteration instead of this hack?

Reply:
there are 2 result sets. not sure why this one is there at all. maybe it's a copy'n'paste error, but I am not entirely sure. so I didn't want to remove it just in case it has a role in testing.

Reply:
This redundant result set was added in #793. @ideoma do you recall if that's on purpose or not?

Reply:
@puzpuzpuz sorry, I don't understand the question, what is on purpose? My memory is not fresh about a PR from 4 years ago.

Reply:
@ideoma there is a superfluous `statement.executeQuery();` in the test. with no assignment or anything. Apparently, for no good reason. 

tl;dr: Don't worry, I will remove it. I was just overly cautious.